misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -3886,6 +3904,17 @@ private <T extends Node> boolean isMultilineModuleMember(T node) {
         }
     }
 
+    private <T extends Node> boolean isMemberOfScope(T node) {
+        switch (node.kind()) {
+            case OBJECT_METHOD_DEFINITION:
```

Comment:
What is the syntax kind we are getting for remote method definition? Add a test case to cover remote methods as well.

Reply:
In the case of remote methods the syntax kind is `RESOURCE_ACCESSOR_DEFINITION`. Will add a test case for that